calyx-py/calyx/builder.py

```diff
@@ -463,6 +500,12 @@ def not_(self, size: int, name: str = None) -> CellBuilder:
         name = name or self.generate_name("not")
         return self.logic("not", size, name)
 
+    def const_mult(self, size: int, const: int, name: Optional[str] = None) -> CellBuilder:
+        """Generate a StdConstMult cell."""
+        name = name or self.generate_name("const_mult")
+        self.prog.import_("primitives/binary_operators.futil")
+        return self.cell(name, ast.Stdlib.const_mult(size, const))
```

Comment:
Nice! I was wondering why you didn't just use `binary`, but I'm guessing that the AST's `op` is not good enough for you: you need to pass this `const` business too. If you think this will be a common pattern, feel free to pipe it all the way through the AST, something like:

```
    @staticmethod
    def op(op: str, bitwidth: int, signed: bool):
        return CompInst(f'std_{"s" if signed else ""}{op}', [bitwidth])

    @staticmethod
    def const_op(op: str, bitwidth: int, const: int, signed: bool):
        ...

```

Reply:
If this comes up again will definitely generalize, but for now this is the only case I've had for a const_op type thing